### PR TITLE
Add pin view

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

File: src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java [hunk 1]
```diff
@@ -44,4 +44,16 @@ public String psuDataInput(@RequestBody PsuData psuData, HttpSession session) {
         // TODO change to the appropriate view
         return "";
     }
+
+    @GetMapping("/pin")
```
Comment (spetrychenko) [inline src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java]: we should keep the same convention for page URL strategy, please have a look at previous pages.

File: src/main/resources/templates/pin.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+          integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <title>Pin page</title>
+</head>
+<body>
+    <div class="container pt-5 pb-5 d-md-flex justify-content-center">
+        <form method="post" action="/pin" class="form-inline">
+            <div class="form-group raw flex-nowrap" style="width: 65vh; justify-content: space-around">
+                <label for="pin" class="mr-3 h5">Please enter your PIN:</label>
+                <input name="pin" id="pin" type="text" class="form-control mr-3 col-sm-5">
```
Comment (spetrychenko) [inline src/main/resources/templates/pin.html]: should be type password

File: src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java (hunk 1 above)
Comment (berezkin88) [reply src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java]: done

File: src/main/resources/templates/pin.html (hunk 2 above)
Comment (berezkin88) [reply src/main/resources/templates/pin.html]: done

**Pull request merged**